addons/website_sale/models/res_config_settings.py

```diff
@@ -37,9 +31,9 @@ def _default_recovery_mail_template(self):
     module_account = fields.Boolean("Invoicing")
 
     cart_recovery_mail_template = fields.Many2one('mail.template', string='Cart Recovery Email', domain="[('model', '=', 'sale.order')]",
-                                                  default=_default_recovery_mail_template, related='website_id.cart_recovery_mail_template_id', readonly=False)
+                                                  related='website_id.cart_recovery_mail_template_id', readonly=False)
```

Comment:
This field is `readonly=False`, why do you remove the default?  Or is it a more general bugfix?

Reply:
See #67774